### PR TITLE
App Submission: Baikal - a Calendar + Contacts server 

### PR DESCRIPTION
**Pull request by dennysubke**
# App Submission

### Baikal

h3B1DaG - Imgur

### 

Baikal Server is a lightweight CalDAV and CardDAV server designed for users who want to manage their calendar and contact information securely on their own server. This robust application offers an extensive web interface that simplifies the management of users, address books, and calendars, making it accessible for both beginners and experienced users.

###
InOvJFa - Imgur

hIQ1ZIv - Imgur

nobRqdE - Imgur

### I have tested my app on:
- [ ] umbrelOS on a Raspberry Pi
- [X] umbrelOS on an Umbrel Home
- [ ] umbrelOS on Linux VM

### CODE REVIEW

**Review by dennysubke — COMMENTED** [review]
- The app_proxy service has been set up to forward requests to the Baikal web service (`baikal_web_1`) on port 80. 
- Updated volume mappings for Baikal's configuration and data directories.
- Removed the commented-out volume `${APP_DATA_DIR}/data/baikal:/var/www/html/Specific` and replaced it with the correct path.

**Comment by dennysubke** [comment]
> Thanks for another submission @dennysubke. I have left a review and suggestions below. Once addressed I can give it a test.

Done. I hope I haven't forgotten anything. :)

File: baikal/umbrel-app.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+manifestVersion: 1
+id: baikal
+name: Baikal
+tagline: A lightweight CalDAV+CardDAV server
+icon: ""
```
Comment (nmfretz) [inline baikal/umbrel-app.yml:5]: Sorry for the confusion here. This entire line `icon: ""` can be completely removed.

**Comment by nmfretz** [comment]
> Done. I hope I haven't forgotten anything. :)

Thanks @dennysubke. I have removed the unused named volume, since we are using bind mounts for data persistence: https://github.com/getumbrel/umbrel-apps/pull/1606/commits/0c351f2da0083ecbf09af724252318020e08f524

I've also pushed some additional commits to finalize this submission: https://github.com/dennysubke/umbrel-apps/compare/85656d1ae7d5b1871c0d5d25c4c41239b737aa4f...f4453924c2e9756659a233c06aa808cacd9827a1 

I have tested and it is working well for me. Going live to the app store now. Thanks again for your efforts bringing this to the official app store 🎉

image

**Comment by Niclassslua** [comment]
image

<br><br>

Compared to other App Icons the Baikal icon is very low quality.

I would suggest using the one I uploaded. It's 900x900.

<br><br>

baikal

**Comment by dennysubke** [comment]
@Niclassslua https://github.com/getumbrel/umbrel-apps/pull/1538#issuecomment-3267864983

**Pull request merged**